admiral/pkg/apis/admiral/routes/handlers.go

```diff
@@ -1,17 +1,30 @@
 package routes
 
 import (
+	"encoding/json"
 	"fmt"
+	"github.com/gorilla/mux"
 	"github.com/istio-ecosystem/admiral/admiral/pkg/clusters"
+	"istio.io/client-go/pkg/apis/networking/v1alpha3"
 	"log"
 	"net/http"
+	"strings"
 )
 
 type RouteOpts struct {
 	KubeconfigPath string
 	RemoteRegistry *clusters.RemoteRegistry
 }
 
+//type ClusterServiceEntries struct {
+//	ServiceEntries []v1alpha3.ServiceEntry `json:"ServiceEntries,omitempty"`
+//}
+
+type IdentityServiceEntry struct {
+	IdentityCname string   `json:"IdentityCname,omitempty"`
```

Comment:
Can this be changed to just Cname?, IdentityCname seems a bit misleading

Reply:
Sure, added fix for this